### DISCUSSION TITLE
Display bug in account creation date in admin dashboard

### DISCUSSION THREAD

**Discussion by Genai-labs** [open-webui/open-webui#4251] [General]
# Bug Report 

### Bug Summary: 

Date formatting : In the administrator dashboard, there seems to be a problem with the formatting of the creation date of the various accounts added, when the selected language is French. 

### Steps to Reproduce: 

1. Launch Open WebUI  
2. Change language to French 
3. Access the Admin panel > Dashboard  

### Expected Behavior: 

AM or PM should not appear more than once in the creation column, or as in other languages, should not appear at all. 

### Actual Behavior: 

Several AM/PM annotations in the creation column (Appears to be in French only) 

### Environment: 

Open WebUI Version: v0.3.10 

#### EN
![Date](https://github.com/user-attachments/assets/79fee4f8-e6f1-437c-9f13-85e204d02b44)

#### FR
![Date_FR](https://github.com/user-attachments/assets/39acaa1e-bed5-4261-9b0b-cc43a7c2cdc0)

**Comment by tjbck** [comment]
Seems to be a translation issue, PR welcome!
